packages/koa-shopify-webhooks/src/register.ts

```diff
@@ -4,6 +4,7 @@ import {WebhookHeader, Topic} from './types';
 export enum ApiVersion {
   April19 = '2019-04',
   July19 = '2019-07',
+  October1 = '2019-10',
```

Comment:
wait should this be `October19` for both of these to match the others?

Reply:
Wow yes. This is why you don't do any work at 6 am 🙃I'll update.